pkgs/applications/audio/open-music-kontrollers/generic.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, lib, fetchurl, pkg-config, meson, ninja, libGLU, lv2, serd, sord, libX11, libXext, glew, lv2lint
+, pname, version, sha256, description
+, url ? "https://git.open-music-kontrollers.ch/lv2/${pname}.lv2/snapshot/${pname}.lv2-${version}.tar.xz"
+, additionalBuildInputs ? []
+, postPatch ? ""
+, ...
+}:
+
+stdenv.mkDerivation {
+  inherit pname;
+
+  inherit version;
+
+  inherit postPatch;
+
+  inherit description;
+
+  src = fetchurl {
+    url = url;
```

Comment:
Some of them at least can probably use fetchGit which should probably be used instead.

Reply:
How should I do that in this situation?

Reply:
I am not sure tbh because the sources are all a bit different. Maybe we can pass in src instead of url and sha256?

Reply:
What is the downside of keeping it like this?
I find it quite elegant. :)

Reply:
How do these snapshots work? Do they need to be generated by hand? if so they could disappear and not exist for every commit so updating to another git rev would be unnecessarily hard.

Reply:
They are automatically generated.
Does that make my code OK, or did I misunderstand?

Reply:
Then it might be fine. Not fully sure to be honest.

Reply:
Is this in an ok state now?